llvm/test/CodeGen/AArch64/loop-dependence-mask-ccmp.ll

```diff
@@ -1,45 +0,0 @@
-; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
-; RUN: llc -mtriple=aarch64 -mattr=+sve2 -verify-machineinstrs -stop-after=finalize-isel %s -o - | FileCheck %s
-
-; Regression test for a bug where getTargetConstant(0) was used instead of
-; getConstant(0) in ScalarizeVecRes_LOOP_DEPENDENCE_MASK, causing instruction
```

Comment:
Note: I removed this test as what it is testing does not occur anymore (and as as seen from the comment was a minor mix-up between getTargetConstant and getConstant).